blocks/init.php

```diff
@@ -0,0 +1,238 @@
+<?php
+/**
+ * Blocks Initializer
+ *
+ * Enqueue CSS/JS of all the blocks.
+ *
+ * @since   1.0.0
+ * @package CGB
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+
+/**
+ * Register the block on the init hook.
+ *
+ * Note: We need to pass the attributes in order for the ServerSideRender to work
+ */
+function register_author_avatars_block() {
```

Comment:
I deleted the class that was handling this. This is much simpler but feel free to move it around as needed.